go.sum

```diff
@@ -211,17 +207,13 @@ github.com/uber/jaeger-lib v2.4.1+incompatible h1:td4jdvLcExb4cBISKIpHuGoVXh+dVK
 github.com/uber/jaeger-lib v2.4.1+incompatible/go.mod h1:ComeNDZlWwrWnDv8aPp0Ba6+uUTzImX/AauajbLI56U=
 github.com/uber/ringpop-go v0.8.5/go.mod h1:zVI6eGO6L7pG14GkntHsSOfmUAWQ7B4lvmzly4IT4ls=
 github.com/uber/tchannel-go v1.22.2/go.mod h1:Rrgz1eL8kMjW/nEzZos0t+Heq0O4LhnUJVA32OvWKHo=
-github.com/uber/tchannel-go v1.22.3 h1:rmIIlBLM2gvel//NxNWvhxvtbtMSeu68v3T95KZELCs=
```

Comment:
weird. why would x/net result in a tchannel cleanup? and just one minor version as opposed to all?
Did we just not cleanup earlier?